src/app/ticker/actions.js

```diff
@@ -31,16 +31,35 @@ export type Action =
 
 // eslint-disable-next-line
 export const register = () => async (dispatch: Function, getState: GetState) => {
-  dispatch(ui.txStart('Submitting token symbol registration...'))
-  try {
-    const details: SymbolDetails = getState().form[formName].values
-    await TickerRegistry.registerTicker(details)
-
-    dispatch(ui.txEnd(null))
-    dispatch(registered())
-  } catch (e) {
-    dispatch(ui.txFailed(e))
-  }
+  dispatch(
+    ui.showModal(
+      'Before you proceed with your Token Symbol Reservation',
+      'Confirmation Required',
+      'Reserve Ticker',
```

Comment:
My mistake. I was referencing an older design. I can update the ui component accordingly since that doesn't change either (including 'Confirmation Required')